modules/styling-transform/spec/createProgramFromSource.ts

```diff
@@ -96,7 +103,10 @@ export function createProgramFromSource(...args: any[]) {
     getDirectories: path => ts.sys.getDirectories(path),
     // This should be kept up to date with getSourceFile()
     fileExists: fileName => {
-      return !!sourceFiles.find(s => s.fileName === fileName) || ts.sys.fileExists(fileName);
+      return (
+        !!sourceFiles.find(s => s.fileName === getLocalFileName(sources, fileName)) ||
```

Comment:
```suggestion
        sourceFiles.some(s => s.fileName === getLocalFileName(sources, fileName)) ||
```

Can we use `.some` here as we are looking for a boolean value?